### ISSUE TITLE
Class names slow to load

### ISSUE THREAD

**Issue by CarsonCook** [QueensEngineeringSociety/QLife#24]
Names are loading only on the first time that day of the week is selected for each login.

So login, slow to load Monday, but the next monday is fine once 1st once loaded. Can kill app and come back and they are there, only happens when schedule re-downloaded.

**Comment by ruffoa** [comment]
Right, that's how it's currently set up, it does a one-time asynchronous download into the database after the ICS file has been downloaded.  The other option would be to take a little longer in the initial setup, and wait until the names are downloaded prior to starting the dayView for the first time

**Comment by CarsonCook** [comment]
Yeah, I figured.

I think it's better to do it all at once in the initial setup
